src/app/components/SwapWidget/style.scss

```diff
@@ -0,0 +1,107 @@
+.container {
```

Comment:
Rather than using custom css here I would use the reactstrap Card so the design is consistent. If you don't like how cards look feel free to tweak it in faast-ui